Packages/com.unity.inputsystem/InputSystem/Devices/Gamepad.cs

```diff
@@ -154,18 +151,18 @@ public struct GamepadState : IInputStateTypeInfo
         ////REVIEW: should left and right trigger get deadzones?
 
         /// <summary>
-        /// Position of the left trigger. Goes from 0 (not pressed) to 1 (fully pressed).
+        /// The current position of the left trigger on a gamepad.
         /// </summary>
-        /// <value>Position of left trigger.</value>
+        /// <remarks>The value's range goes from 0 to 1, where 0 represents the trigger not pressed at all, and 1 represents the trigger in its fully pressed position.</remarks>
```

Comment:
not pressed at all -> in it's neutral position, its -> it's

Reply:
likewise in other parts of the code.

Reply:
Fixed